helpers/setup_assist.py

```
@@ -0,0 +1,319 @@
+        label_reg = lr.async_get(hass) if lr is not None else None
+    except Exception as err:
+        _LOGGER.debug(
+            "Setup assist registry lookup failed for %s: %s",
+            clean_entity_id,
+            err,
+        )
+        return unsupported_setup_assist_suggestion("registry_lookup_failed", clean_entity_id)
+
+    try:
+        entity_entry = entity_reg.async_get(clean_entity_id) if entity_reg is not None else None
+    except Exception as err:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)